src/classes/ADDR_Seasonal_TEST.cls

```diff
@@ -920,12 +920,12 @@ private with sharing class ADDR_Seasonal_TEST {
             MailingStreet__c = '123 Seasonal',
             MailingCity__c = 'Seattle',
             MailingState__c = 'WA',
-            Seasonal_Start_Day__c = String.valueOf(system.today().addDays(-1).day()),
-            Seasonal_Start_Month__c = String.valueOf(system.today().month()),
-            Seasonal_Start_Year__c = system.today().addYears(1).year(),
-            Seasonal_End_Day__c = String.valueOf(system.today().day()),
-            Seasonal_End_Month__c = String.valueOf(system.today().month()), 
-            Seasonal_End_Year__c = system.today().addYears(1).year()
+            Seasonal_Start_Day__c = 1,
+            Seasonal_Start_Month__c = 6,
+            Seasonal_Start_Year__c = system.today().year(),
+            Seasonal_End_Day__c = 31,
+            Seasonal_End_Month__c = 7, 
+            Seasonal_End_Year__c = system.today().year()
```

Comment:
Correct. I read the date incorrectly as the start date being 1//1/{!CurrentYear}. Great work.